apps/test-moisture/src/main.cpp

```diff
@@ -7,35 +7,32 @@ DigitalOut led1(LED1);
 int main() {
   led1 = 0;  // start with LED off
 
-  uint16_t moisture = 0;  // initialize default variables
+  float moisture = 0;  // initialize default variables
   float temperature = 0.0;
-  uint8_t HW_ID     = 0;
 
   MoistureSensor sensor = MoistureSensor(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL);
 
   while (1) {
-    sensor.Reset_Sensor();  // reset all registers on sensor to default values
+    sensor.resetSensor();  // reset all registers on sensor to default values
 
     ThisThread::sleep_for(500ms);
 
-    led1 = (sensor.Is_Initialized());  // turn on LED if the sensor's HW_ID code matches the known value
+    led1 = (sensor.getSensorStatus());  // turn on LED if the sensor's HW_ID code matches the known value
 
     printf("\r\nChecking Device ID...\r\n");  // read device HW_ID, reading of 85 is expected
-    HW_ID = sensor.Read_HW_ID();
-    printf("HW ID: %d \r\n", HW_ID);
```

Comment:
Why did you keep the line that says it's checking the device ID, then remove the device ID check? Since the LED indicates the pass/fail of the check, you may as well remove the print statement (I think).

Reply:
Will change thanks

Reply:
can u click "resolve convo"